packages/primitives/vector-shape/src/types.ts

```diff
@@ -0,0 +1,9 @@
+import { TColor } from 'colorido'
+
+export type TVectorShape = {
+  id?: string,
+  height?: number,
+  path?: string,
```

Comment:
`path: string`? does it make sense to make it optional?

Reply:
Fair enough